File: ios/BarcodeViewManager.m
//
//  BarcodeViewManager.m
//  qrparser
//
//  Created by Bulat on 8/21/24.
//

#import <Foundation/Foundation.h>
#import "React/RCTViewManager.h"
@interface RCT_EXTERN_MODULE(BarcodeViewManager, RCTViewManager)
RCT_EXPORT_VIEW_PROPERTY(onBarcodeRead, RCTDirectEventBlock)
@end

File: ios/BarcodeViewManager.swift
//
//  BarcodeViewManager.swift
//  qrparser
//
//  Created by Bulat on 8/21/24.
//

import Foundation

@objc(BarcodeViewManager)
class BarcodeViewManager : RCTViewManager {
  override func view() -> UIView! {
    return BarcodeView()
  }
  
  override static func requiresMainQueueSetup() -> Bool {
    return true
  }
}
